src/components/JumpIn/JumpIn.tsx

```diff
@@ -35,19 +35,30 @@ const JumpIn = React.memo((props: JumpInProps) => {
       e.preventDefault()
 
       const hasLauncher = await launchDesktopApp(desktopAppOptions)
-      !hasLauncher && setIsModalOpen(true)
 
       onTrack?.({
+        type: hasLauncher ? "JUMP_IN" : "OPEN_DOWNLOAD_MODAL",
+        url: hasLauncher ? undefined : downloadUrl,
         has_launcher: hasLauncher,
       })
+
+      !hasLauncher && setIsModalOpen(true)
     },
-    [onTrack, desktopAppOptions]
+    [onTrack, downloadUrl, desktopAppOptions]
   )
 
   const handleDownloadClick = useCallback(
     (e: React.MouseEvent<HTMLButtonElement>) => {
       e.stopPropagation()
       e.preventDefault()
+
+      // Call onClick for tracking if provided
+      onTrack?.({
+        type: "DOWNLOAD",
```

Comment:
and use the same enum here
```suggestion
        type: JumpInEventType.Download,
```